cfgov/unprocessed/css/organisms/featured-menu-content.less

```diff
@@ -0,0 +1,110 @@
+// Duplicated from current implementation in Capital Framework v4.6.0
+// with minor edits (to be released in CFv4.7.0)
+// This can be deleted safely when this project is ported to CFv4
+
+@fcm-bp-max: 700px;
+@fcm-bp-min: 701px;
+
+.o-featured-content-module {
+  .webfont-regular();
+
+  position: relative;
+
+  min-height: 320px;
+  margin-right: -1px;
+  margin-left: -1px;
+
+  background-color: @block__bg;
+
+  .respond-to-max( @bp-xs-max, {
+    padding-right: unit( 15px / @base-font-size-px, em);
+    padding-left: unit( 15px / @base-font-size-px, em);
+  } );
+
+  .respond-to-range( @bp-sm-min, @fcm-bp-max, {
+    padding-right: unit( 30px / @base-font-size-px, em);
+    padding-left: unit( 30px / @base-font-size-px, em);
+  } );
+
+  &_text {
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em);
+    padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em);
+
+    .respond-to-min( @fcm-bp-min, {
+      width: 300px; // width of 5 of 12 cols at 799px - 30px padding
+
+      padding-right: unit( @grid_gutter-width / @base-font-size-px, em);
+      padding-left: unit( @grid_gutter-width / @base-font-size-px, em);
+    } );
+  }
+
+  &_visual {
+    .respond-to-min( @fcm-bp-min, {
+      position: absolute;
+      left: 360px;
```

Comment:
I know this is a large ask but is there to move away from using position absolute and specifying the left values.